fidesctl/src/fidesctl/core/evaluate.py

```diff
@@ -515,29 +539,40 @@ def evaluate(
     Local Policy definition files will be used as opposed to their
     server-definitions if available.
     """
-    taxonomy = parse(manifests_dir)
 
-    # Populate all of the policies to evaluate
-    taxonomy.policy = get_evaluation_policies(
-        local_policies=taxonomy.policy,
-        evaluate_fides_key=fides_key,
+    # Merge the User-defined taxonomy & Default Taxonomy
+    user_taxonomy = parse(manifests_dir)
+    taxonomy = Taxonomy.parse_obj(
+        {
+            **DEFAULT_TAXONOMY.dict(exclude_unset=True),
+            **user_taxonomy.dict(exclude_unset=True),
+        }
+    )
+
+    # Determine which Policies will be evaluated
+    taxonomy.policy = populate_evaluation_policies(
+        policies=taxonomy.policy,
+        policy_fides_key=policy_fides_key,
         url=url,
         headers=headers,
+        local=local,
     )
-    validate_policies_exist(policies=taxonomy.policy, evaluate_fides_key=fides_key)
-    validate_supported_policy_rules(policies=taxonomy.policy)
-
     echo_green(
-        "Evaluating the following policies:\n{}".format(
+        "Evaluating the following Policies:\n{}".format(
             "\n".join([key.fides_key for key in taxonomy.policy])
         )
     )
     print("-" * 10)
 
     echo_green("Checking for missing resources...")
+    missing_resources = get_referenced_missing_keys(taxonomy)
+    if local and missing_resources:
+        echo_red(str(missing_resources))
+        echo_red("Not all referenced resources exist locally!")
```

Comment:
Would this output be more user-friendly if these lines were swapped, and the resource list indented?

Reply:
It dumps out as a stringified python list, so at least imo its easy to visually parse